### ISSUE TITLE
Copy trip patterns between routes

### ISSUE THREAD

**Issue by wkulesza** [conveyal/gtfs-editor#268]
For a new route, I would like to be able to see other routes’ trip patterns in its dropdown. Currently trip patterns are solely linked to one route.
Is it easily achievable?

**Comment by wkulesza** [comment]
@abyrd @demory sorry guys, i know this is not maitained - but maybe you know if such feature is "hidden" in gtfs editor or in fact available in data tools ui ?

**Comment by abyrd** [comment]
@landonreed do you have any ideas about the above question?

**Comment by landonreed** [comment]
I don't think this exists in gtfs-editor and there isn't really a way to do this in datatools-ui.  I haven't done much work in the gtfs-editor, so I don't really know the level of effort needed to add the feature.
